.eslintrc.yml

```diff
@@ -1,5 +1,7 @@
 env:
   node: true
-extends: "eslint-config-eslint"
+# extends: "eslint-config-eslint"
```

Comment:
it will disable all rules in the config. can we just use `npm install --force` for the time being - utils we fixed the issue in `eslint-config-eslint`?

Reply:
That didn’t work when I tried. I’d just like to get unblocked. We will be retiring this utility soon so I’m not too concerned if linting is unavailable for a bit.